json/src/value.rs

```diff
@@ -1001,3 +1001,15 @@ pub fn from_value<T>(value: Value) -> Result<T, Error>
     let mut de = Deserializer::new(value);
     de::Deserialize::deserialize(&mut de)
 }
+
+/// A trait for converting values to JSON
+pub trait ToJson {
+    /// Converts the value of `self` to an instance of JSON
+    fn to_json(&self) -> Value;
+}
+
+impl<T: ?Sized> ToJson for T where T: ser::Serialize {
+    fn to_json(&self) -> Value {
+        to_value(&self)
+    }
+}
```

Comment:
It looks like there isn't a trailing newline at the end of the file. Could you add one?